Programowanie Obiektowe/1.2Klasy/_1_3PolaKlasyZad2/Projekt.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _1_3PolaKlasyZad2Zad3
+{
+    internal class Project
+    {
+        private int ProjectId { get; set; }
+        public string ProjectName { get; set; }
+        public List<Worker> Workers { get; set; }
```

Comment:
Moim zdaniem lepiej w polach klasy robić array zamiast List. Jak już musimy mieć jakąś kolekcję a nie array to uzyj IEnumerable:
```suggestion
        public Worker[] Workers { get; set; }
```
```suggestion
        public IEnumerable<Worker> Workers { get; set; }
```

Reply:
Wybrałem opcję z IEnumerable, ponieważ wtedy liczba pracowników pracujących przy projekcie może sie dynamicznie zmieniać.
[0cf34a4](https://github.com/broniuu/Praktyki2022/pull/2/commits/0cf34a4c4ef4bfa47de92441fca3b6da2680db19)